src/sidebar/components/DependencyLessToastMessages.tsx

```diff
@@ -0,0 +1,153 @@
+import {
+  Card,
+  Link,
+  CancelIcon,
+  CautionIcon,
+  CheckIcon,
+} from '@hypothesis/frontend-shared/lib/next';
+import classnames from 'classnames';
+
+import type { ToastMessage } from '../store/modules/toast-messages';
```

Comment:
This type will need to move to a shared location. As a general principle, `src/annotator/` should not depend on anything from `src/sidebar/` and vice-versa. Both may depend on code from `src/shared/` and `src/types/`. We don't have anything in place to enforce the dependency structure between folders at present.